test/k8s/datapath_configuration.go

```diff
@@ -463,81 +449,53 @@ var _ = Describe("K8sDatapathConfig", func() {
 			res.ExpectSuccess()
 			tmpEchoPodPath = strings.Trim(res.Stdout(), "\n")
 			kubectl.ExecMiddle(fmt.Sprintf("sed 's/NODE_WITHOUT_CILIUM/%s/' %s > %s",
-				helpers.GetFirstNodeWithoutCilium(), echoPodPath, tmpEchoPodPath)).ExpectSuccess()
+				"k8s3", echoPodPath, tmpEchoPodPath)).ExpectSuccess()
```

Comment:
Why this change?

Reply:
I'm not sure. This change comes from your suggestion. https://github.com/cilium/cilium/pull/20137#issuecomment-1187125760